src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/CrystalNucleusProfitPer.kt

```diff
@@ -0,0 +1,66 @@
+package at.hannibal2.skyhanni.features.mining.crystalhollows
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.api.event.HandleEvent.Companion.HIGH
+import at.hannibal2.skyhanni.events.mining.CrystalNucleusLootEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.CollectionUtils.sortedDesc
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.NEUItems.getPrice
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.NumberUtil.shortFormat
+
+@SkyHanniModule
+object CrystalNucleusProfitPer {
+    private val config get() = SkyHanniMod.feature.mining.crystalNucleusTracker
+
+    val robotParts = listOf(
+        "CONTROL_SWITCH",
+        "ELECTRON_TRANSMITTER",
+        "FTX_3070",
+        "ROBOTRON_REFLECTOR",
+        "SUPERLITE_MOTOR",
+        "SYNTHETIC_HEART",
+    )
```

Comment:
you only use this list as internal names. instead of always running .asInternalName when using it, simply do `.map { it.asInternalName() }` when you initialize the list